extensions/rich_text_components/Video/Video.py

```diff
@@ -34,6 +34,9 @@ class Video(base.BaseRichTextComponent):
     description = 'A YouTube video.'
     frontend_name = 'video'
     tooltip = 'Insert video'
+    is_block = True
+    preview_url_template = 'https://img.youtube.com/vi/' + \
```

Comment:
Instead of using \ (which is not great, since you don't know what it's escaping, since e.g. there may be a space after it and that's not easy to detect), we use "(...)" instead:

```
    preview_url_template = (
        'https://img.youtube.com/vi/'
        '<[video_id]>/hqdefault.jpg')
```